ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+Provide a description of your issue.
+For any suggestion, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
```

Comment:
Maybe:

`Please put suggestions on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).`

Reply:
Maybe? What dont you like in my phrasing?

Reply:
It's probably fine either way (though 'suggestion' should probably be 'suggestions').